ts/project-details.ts

```diff
@@ -66,7 +67,21 @@ function reload(definition: TableDefinition<any, any>) {
   new TableBuilder(
     definition,
     tableContainerId,
-    getSearchParams(),
-    updateUrlParams
+    handleSearch(),
+    handleFilters
   ).build();
 }
+
+function handleSearch() {
+  const searchParams: Array<Pair<string, string>> = getSearchParams();
+  projectSummaryTable.addFilters(searchParams);
+  // for (const pair of searchParams) {
+  //   projectSummaryTable.applyFilters(pair.key, pair.value, true);
+  // }
+  return searchParams;
+}
```

Comment:
You can use `getSearchParams()` as the third parameter of both `TableBuilder`s to have the same effect. Then you don't need this function or `TableBuilder#addFilters`